data/json/snippets/effect_on_conditions.json

```diff
@@ -0,0 +1,40 @@
+[
+  {
+    "type": "snippet",
+    "category": "NIGHT_MESSAGES_1",
+    "text": [
+      "You see what seems to be a new constellation in the sky.  The stars are quite dense and are precisely aligned in several regular polyhedrons next to each other.",
+      "You see what seems to be a new constellation in the sky.  The stars are in a precise spiral, with the central star constantly shifting color.",
+      "You see several shooting stars.  They seem larger than normal, and are all a deep purple.",
+      "You see a shooting star.  It is bright red and does not fall in a straight line.",
```

Comment:
Do they always IRL look like they're falling in a straight line?

Reply:
AFAIK yes, though one option is "straight toward you".